RELEASE_NOTES.md

```diff
@@ -2,6 +2,11 @@
 
 This section contains past updates we've sent to customers via Discord.
 
+## 1.1.7 (May 24, 2024)
+
+- Updates patch command to prompt for release version.
+- Adds the ability to sign patches. Read more at https://docs.shorebird.dev/patch-signing
```

Comment:
I suspect this is https://docs.shorebird.dev/guides/patch-signing/?

Reply:
https://github.com/shorebirdtech/docs/pull/230 has it as a top-level page. I could see it going either way—the page contains both a guide and an explanation of the feature 🤷

Reply:
lmk if you want me to move the page to be under /guides. I don't feel strongly either way.

Reply:
This guide is eventually the "do it manual" guide, which will end up an enterprise-only feature.  The common case will be "let Shorebird do it for me" which won't even need a doc.  So I'd suggest it be in "less common" places rather than more.  Not sure which that is, but probably under guides? 🤷